test/update-contract-ids.sh

```diff
@@ -37,8 +56,13 @@ grep --include \*.sw -Hno "// AUTO-CONTRACT-ID" . -R | while read line ; do
 
         if [[ $CONTRACT_ID ]]; then 
             popd >> /dev/null
-            sed -i "${LINE}s/0x[a-zA-Z0-9]*/$CONTRACT_ID/g" $FILE
-            echo -e "    ${BOLD_GREEN}ok${NC} ($CONTRACT_ID)"
+
+            if [[ "$INTERACTIVELY" == "0" ]]; then
+              sed -i "${LINE}s/0x[a-zA-Z0-9]*/$CONTRACT_ID/g" $FILE
+              echo -e "    ${BOLD_GREEN}ok${NC} ($CONTRACT_ID)"
+            else
+              echo "$FILE" | xargs -o vim -n -c ":${LINE}s/0x[a-zA-Z0-9]*/$CONTRACT_ID/gc" -c ":x"
```

Comment:
not everyone has vim installed, is there any way that this could be setup to use `$EDITOR` the same way that git does?

Reply:
We can.... but the problem would be the rest of the parameters... I can keep them to try to catch "vi", "vim" etc... but it will not work with any editor. :/

Reply:
For example, the "interactiveness" does not work with "vi".

Reply:
I will try using diff instead.

Reply:
Now I am doing the interactivity without an editor. Should work on more machines.